### ISSUE TITLE
Poor Performance on Waymo Dataset: High Loss and Inadequate Rendering

### ISSUE THREAD

**Issue by Valentina0127** [kcheng1021/GaussianPro#58]
I encountered significant performance issues while running Gaussianpro on the Waymo dataset. Despite following the provided instructions and using the recommended hyperparameters, the results were disappointing. The training loss remained above 0.5, and the rendered images did not exhibit any discernible geometric information of the streets. Instead, they appeared to be composed merely of Gaussian blobs.

Execute the following script:
`#!/bin/bash

# Define paths
DATA_PATH="/path/to/data"  # Replace with the actual data path
SAVE_PATH="/path/to/save"  # Replace with the actual save path

# First training
python train.py -s $DATA_PATH -m $SAVE_PATH \
                --eval --position_lr_init 0.000016 --scaling_lr 0.001 --percent_dense 0.0005 --port 1021 --dataset waymo 

# Render
python render.py -m $SAVE_PATH

# Compute metrics
python metrics.py -m $SAVE_PATH

# Second training
python train.py -s $DATA_PATH -m $SAVE_PATH \
                --eval --flatten_loss --position_lr_init 0.000016 --scaling_lr 0.001 --percent_dense 0.0005 --port 1021 --dataset waymo \
                --sky_seg --normal_loss --depth_loss --propagation_interval 30 --depth_error_min_threshold 0.8 --depth_error_max_threshold 1.0 \
                --propagated_iteration_begin 1000 --propagated_iteration_after 12000 --patch_size 20 --lambda_l1_normal 0.001 --lambda_cos_normal 0.001

# Render
python render.py -m $SAVE_PATH

# Compute metrics
python metrics.py -m $SAVE_PATH
`
